### PR TITLE
Datawrkz Bid Adapter: initial adapter release

### PR DESCRIPTION
**Pull request by vishal-dw**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- Datawrkz Bidder Adapter

Use ```datawrkz``` as bidder

Params

- ```site_id``` required, site ID
- ```bidfloor``` optional, minimum bid for ad unit
- ```deals``` optional, list of deals
- ```outstreamType``` optional, for outstream video adunit. Options: inline, slider_top_left, slider_top_right, slider_bottom_left, slider_bottom_right, interstitial_close, and listicle
- ```outstreamConfig``` optional, player settings for outstream video ad unit

AdUnits configuration example
```
var adUnits = [
   {
      code: 'banner-div',
      mediaTypes: {
        banner: {
          sizes: [[300, 250], [300,600]]
        }
      },
      bids: [{
         bidder: 'datawrkz',
         params: {
           site_id: 'site_id',
           bidfloor: 0.5
         }
       }]
   }]
```

- contact email of the adapter’s maintainer: pubops@datawrkz.com
- [x] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at: https://github.com/prebid/prebid.github.io/pull/3933

### CODE REVIEW

**Comment by ChrisHuie** [comment]
@vishal-dw not sure why our tests ran on the initial commit but now are not showing up after the most recent commit?

**Comment by vishal-dw** [comment]
@ChrisHuie Tests have passed now

File: modules/datawrkzBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,633 @@
+import * as utils from '../src/utils.js';
```
Comment (ChrisHuie) [inline modules/datawrkzBidAdapter.js:1]: Can you please import just the needed functions please 🙏

Comment (vishal-dw) [reply modules/datawrkzBidAdapter.js]: Pull updated

**Review by ChrisHuie — APPROVED** [review]

**Pull request merged**